pkg/model/components/kubelet.go

```diff
@@ -243,7 +243,7 @@ func (b *KubeletOptionsBuilder) BuildOptions(o interface{}) error {
 		clusterSpec.Kubelet.FeatureGates = make(map[string]string)
 	}
 	if _, found := clusterSpec.Kubelet.FeatureGates["ExperimentalCriticalPodAnnotation"]; !found {
-		if b.Context.IsKubernetesGTE("1.5.2") {
+		if b.Context.IsKubernetesGTE("1.5.2") && b.Context.IsKubernetesLT("1.16") {
```

Comment:
Hi @rifelpet, was it added to the recent release? I'm trying do start a cluster with kops v. 1.14.0 and kubernetes v1.17.0-alpha.1 or v.1.16.0/1 and getting same error on kubelet:
```
Oct 03 12:05:59 ip-172-20-34-2 kubelet[2601]: F1003 12:05:59.491671    2601 server.go:180] unrecognized feature gate: ExperimentalCriticalPodAnnotation
```
command used to start a cluster:
```
kops create cluster \
         --cloud aws \
         --zones eu-west-1b     \
         --master-zones eu-west-1b   \
         --ssh-public-key $HOME/.ssh/id_rsa.pub    \
         --name=test.example.com \
         --state=s3://test.example.com \
         --kubernetes-version=v1.17.0-alpha.1
```
Am i doing something wrong? Or it's expected and you don't support those versions yet?

Reply:
According to the compatibility matrix on Kops' readme, Kops 1.14 only supports Kubernetes 1.14.X. Until a Kops 1.16 alpha is released (hopefully soon), you'll have to use a build from the master branch to run Kubernetes 1.16, and I wouldn't recommend doing that for production environments until Kops 1.16 stable is released but feel free to give it a shot!

Reply:
Yeah, thank you, gave it a shot just yesterday (Version 1.15.0-alpha.1 (git-447e0cf1e)
) and ended up with a different issue with nodeLabels. Kubelet is yet failed to start :(
```
kubelet[3014]: F1004 09:05:14.262379    3014 server.go:185] unknown 'kubernetes.io' or 'k8s.io' labels specified with --node-labels: [kops.k8s.io/instancegroup kubernetes.io/role
 node-role.kubernetes.io/master]
```
for labels:
```
--node-labels=kops.k8s.io/instancegroup=master-eu-west-1b,kubernetes.io/role=master,node-role.kubernetes.io/master=
```
Apparently it's a different issue with support of some labels in the new kubernetes version, I think that label `node-role.kubernetes.io` is not available anymore or smth